WeatherApp/src/main/java/org/example/Menu.java

```diff
@@ -0,0 +1,46 @@
+package main.java.org.example;
+
+import java.util.Scanner;
+
+public class Menu {
+
+    public static void main(String[] args) {
+
+        Scanner scanner = new Scanner(System.in);
+        String userOptions = "a";
```

Comment:
Dziwna trochę konstrukcja, nie ma potrzeby inicjalizować tej zmiennej konkretną opcją, bo to może wprowadzać w błąd na pierwszy rzut oka.
Zrobiłbym po prostu `userOptions = ""`.

Reply:
:-) dziwna ale działająca ;-)) poprawiam :-)

Reply:
ok